src/instructlab/sdg/utils/taxonomy.py

```diff
@@ -21,9 +24,13 @@
 import yaml
 
 # Local
-from . import chunking
+from .chunkers import DocumentChunker
+
+# Initialize the pdf parser
+parser = pdf_parser()
```

Comment:
@khaledsulayman @bbrowning 
do we know if this internally downloads the docling models automatically or would we need to externally ensure that those models are present before invoking the parsing?

Reply:
I would like to ensure that we error out if those models are not available on disk rather than silently download them somehow